src/Compiler/TaskTableView.cpp

```diff
@@ -292,6 +292,16 @@ void TaskTableView::addTaskLogAction(QMenu *menu, FOEDAG::Task *task) {
       menu->addAction(viewReport);
     }
   }
+
+  // create custom interactive path analysis action
+  if (task->title() == "Timing Analysis") {
```

Comment:
Please check by id but not title. It can change any time.

Reply:
> Please check by id but not title. It can change any time. `m_taskManager->taskId((*it)) == TIMING_SIGN_OFF`
corrected